Public/Helpers/mathHelper.js

```diff
@@ -0,0 +1,3 @@
+export function randomIntFromInterval(min, max) {
```

Comment:
You shouldn't have any .js files in the repo since these will be the compiled files

Reply:
Changed the .gitignore file to ignore those files